guide/payments/receive.md

```diff
@@ -0,0 +1,219 @@
+
+The following are the standard properties of a payment link:
+
+- **Address** – The destination of the payment
+- **Amount** – How much is being requested (optional)
+- **Label** – For example, the name of who is sending the payment (optional)
+- **Message** – For example, the purpose of payment (optional)
+
+Not all applications support storing or even sharing payment links with these details. When providing this functionality, find the right balance between convenience and usefulness for your target audience. Try to also think of ways that you can make inputting these details take less time, so it isn't burdensome.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/enter-details-amount.png"
+   retina = "/assets/images/guide/payments/receive/enter-details-amount@2x.png"
+   alt-text = "Screen asking the receiver to enter the amount being requested"
+   caption = "The receiver can specify the amount that they are requesting. This could be shared in payment links which would auto-fill the amount for the sender."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/enter-details-from.png"
+   retina = "/assets/images/guide/payments/receive/enter-details-from@2x.png"
+   alt-text = "Screen asking the receiver to enter the name of the sender"
+   caption = "The receiver can use a label to specify who the payment is from. This can be stored and used in suggestions when making future payment requests."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/enter-details-purpose.png"
+   retina = "/assets/images/guide/payments/receive/enter-details-purpose@2x.png"
+   alt-text = "Screen asking the receiver to enter the purpose of the payment"
+   caption = "Sometimes the purpose can be determined by identifying who the payment is from. Be mindful of that, and consider easy ways to help the receiver add more context."
+   width = 250
+   height = 541
+%}
+
+</div>
+
+<!--
+Technical nitpick;
+Addresses are not exactly "generated"
+since they are deterministic technically
+what happens is an unused address just gets chosen
+-->
+
+<!-- 
+Update interopability page
+Since there are many Bitcoin applications for people to choose from, its likely that the sender and receiver are not using the same one and they don't have the abilitiy to exchange payment details within the application itself. 
+
+The exchange of payment details then most commonly happens outside of the wallet application. This 
+
+Since bitcoin is a open system and has many payment applications built ontop of it, there is a [standard format]({{ '/guide/foundations/wallet-interoperability/#payment-links' | relative_url }}) of "payment links" that most bitcoin applications use to share payment details.
+-->
+
+## Sharing the payment request
```

Comment:
```suggestion
## Sharing the request
```

Reply:
also with this one, i think its just clear to have the payment part in here. the payment request concept is something that will be in other places in this chapter I expect so might as well readers familiar with it.

Reply:
You decide. I think it's pretty clear from the page context that it's about payments but up to you.